templates/springboot/http/src/main/java/functions/CloudFunctionApplication.java

```diff
@@ -13,8 +15,22 @@ public static void main(String[] args) {
   }
 
   @Bean
-  public Function<String, String> uppercase() {
-    return String::toUpperCase;
-  }
+  public Function<Message<String>, String> echo() {
+    return (inputMessage) -> {
 
+      var stringBuilder = new StringBuilder();
```

Comment:
this does a lot, shouldn't we just echo body

Reply:
@mcruzdev  ^^^

Reply:
otherwise you must fix expected in e2e test

Reply:
or maybe not if there are no headers...

Reply:
the test is passing

Reply:
it's ok I guess

Reply:
It is the same thing that `go:http` does.

Reply:
> 

I would like to add `echo:  message=hello` for expects